client/landing/subscriptions/components/site-subscriptions-manager/site-row.tsx

```diff
@@ -193,6 +207,16 @@ const SiteRow = ( {
 		recordPostEmailsSetFrequency( { blog_id, delivery_frequency } );
 	};
 
+	const newColumnIcon = useMemo(
```

Comment:
If you prefer to keep as a function, so maybe this variable could be renamed to `newCommentIcon` to be clearer.

Reply:
I'll change it to component 👍